spec/swagger/definitions/court_hearing.json

```diff
@@ -69,7 +69,7 @@
         "properties": {
           "move": {
             "$ref": "#/definitions/move_reference/MoveReference",
-            "description": "Reference to the move entity that is associated with this court hearing."
+            "description": "This is required for us to create a Court Hearing in nomis. Reference to the move entity that is associated with this court hearing."
```

Comment:
I am not sure about using the term "required",  this could make the users think that it is required, but actually you can call the endpoint without a move.
What about something like:

"Reference to the move entity that is associated with this court hearing. If it is present, Nomis will be updated with the new Court Hearing."